### PR TITLE
Inprogress

### PR DESCRIPTION
**Pull request by imypullan**
Finalised media queries (so many media queries)

### CODE REVIEW

**Comment by dotNATE** [comment]
I really like the projects section. Couldn't find any red flags in the code so mostly g2g.

If you make the screen about 300-320px wide then your contact and about me sections get a little bent out of shape, otherwise all in working order.

File: style.css [hunk 1]
```diff
@@ -154,27 +153,38 @@ header div {
...
 }
 
 .item h3, .item p {
 	display: none;
 }
 
+.item h3 {
+	margin-top: 5px;
+	font-size: 2rem;
+	margin-bottom: 0px;
```
Comment (dotNATE) [inline style.css:185]: margin-bottom: 0px; - could this be removed?

File: style.css [hunk 2]
```diff
@@ -299,29 +305,58 @@ a:focus
 	transition-duration: 500ms;
 }
 
+.projects a,
+.projects a:hover,
+.projects a:focus {
+	color: #000;
+	transition-duration: none;
```
Comment (dotNATE) [inline style.css:312]: Is this not the default value? If not, great; if so, can it be removed?

File: style.css [hunk 3]
```diff
@@ -299,29 +305,58 @@ a:focus
...
-@media only screen and (max-width: 900px) {
+@media only screen and (max-width: 1000px) {
 	
 	.container {
 		grid-template-columns: 1fr 1fr;
 		margin: 0 auto;
 	}
 	
+	.item {
+		margin: 20px 20px;
```
Comment (dotNATE) [inline style.css:332]: Margin: 20px; would be the same, I think.

File: style.css (hunk 2 above)
Comment (imypullan) [reply style.css:312]: One was default; removed thanks

**Pull request merged**